Cogs/StudentCommands.py

```diff
@@ -28,15 +28,12 @@ async def corgme(self, ctx, number=-1):
         # Get images from directory
         images = ['dogs/corgis/' + path.name for path in Path('dogs').rglob('*.*')]
         print(images[0])
-
-        # Pick a random image
-        if number != -1 and (0 < number < len(images)):
-            image = images[number]
-        else:
-            image = images[randint(0, len(images) - 1)]
-
+        #Generates a random number if no number is given
```

Comment:
Please put a new line between 30 and 31, and a space after # in 31

Reply:
alright I think I got it